### ISSUE TITLE
changes required to poll...

### ISSUE THREAD

**Issue by despiegk** [threefoldfoundation/home#69]
### decentralization version nr support

![image](https://user-images.githubusercontent.com/6021844/88136750-3af2d500-cbea-11ea-9b49-870f220ed12f.png)

```decentralization manifesto -> decentralization manifesto v2.0.1```

in the vote we need to remember which manifesto version used !, change your db records to track 2.0.0 till today, from today we track 2.0.1

![image](https://user-images.githubusercontent.com/6021844/88137033-f0258d00-cbea-11ea-99f7-3da483929009.png)

add version nr here as well

![image](https://user-images.githubusercontent.com/6021844/88137057-05022080-cbeb-11ea-87cb-e4a68a618d3b.png)

```
Thank you for confirming our "Decentralization manifesto", you have now digitally signed this document.
```

needs to become

```
Thank you for confirming our "Decentralization manifesto v2.0.1", you have now digitally signed this document.
```

### kristof/rob access voting machine

Kristof/Rob is a TFGrid council and as such can have access to the voting machine.
Please arrange and tell them privately how to access.
Make sure ONLY TFGrid council has access to the voting results.

### remove irrelevant screens

![image](https://user-images.githubusercontent.com/6021844/88136960-c2d8df00-cbea-11ea-9110-62a078355c63.png)
and the percentage per choice

### TFTA voting rights

TFTA on Stellar has all the same rights and more compared to the TFT on Rivine.

change
```
TFTA on Stellar has all the same rights and more compared to the TFT on Rivine.
I can 
- Buy any capacity on the TF Grid, which represents the main use-case of this token.
- SellTFTA to anyone (directly or using Stellar Exchange or using atomic swaps)
- Transfer TFTA to anyone
Enjoy any other feature you would expect from a digital currency

No rights have been taken away from me by switching blockchains.
```
to
```
May 2020, TFT users could migrate their tokens towards a new public blockchain called Stellar.
Each user had to do this action him/herself. The result of the migration is the ownership of a digital asset called TFTA on the public blockchain Stellar. TFTA on Stellar has all the same rights as the TFT on Rivine. Additional benefits are: Stellar Exchange and a public blockchain with more users so more chance for liquidity.

I can 
- Buy any capacity on the TF Grid, which represents the main use-case of this token.
- Sell TFTA to anyone (directly or using Stellar Exchange or using atomic swaps)
- Transfer TFTA to anyone

No rights have been taken away from me by switching blockchains. Please realize this question has nothing to do in relation to Liquid and BTCAlpha public exchange integration, this is next question.
```
note was space missing between '''sell TFTA'''

### remove the block if they did not sign decentralization...

in other words, everyone can vote

also on the sell bot, allow everyone to sell as long as voting has happened !

**Comment by despiegk** [comment]
also lets change language on result

```

Reading June 2020 update document
I have read the June 2020 update document: 94.9%
I have not read the June 2020 update document: 5.1%
Reading the manifesto
I have read the manifesto on http://decentralization2.threefold.io and I do agree with the contents of this manifesto.: 100.0%
I have not read the manifesto on http://decentralization2.threefold.io or I do not agree.: 0.0%
TFTA on Stellar rights
I do agree: 94.9%
I do not agree: 5.1%
TFTA Availability
I am fine with the option to sell my TFTA (TFTv1) on the Stellar exchange or any other decentralized market mechanism and get automatic conversion to TFTv2 end of the year.: 69.39%
I want my TFTA to be available on supported exchanges as TFT and agree with minimal price protection (0.15 USD, +2% increase per month starting with May 1), sales will happen through a sales bot.: 29.59%
I want my TFTA to be available on supported exchanges as TFT and there should be no price protection. I do realize this choice has the potential to damage the ThreeFold movement.: 1.02%
```

to

```

Reading June 2020 update document
I have read the June 2020 update document: 94.9%
I have not read the June 2020 update document: 5.1%
Reading the manifesto
I have read the decentralization manifesto and I do agree with the contents of this manifesto.: 100.0%
I have not read the decentralization manifesto or I do not agree.: 0.0%
TFTA on Stellar rights
I do agree: 94.9%
I do not agree: 5.1%
TFTA trading mechanism support
I am fine with the option to sell my TFTA (TFTv1) on the Stellar exchange or any other decentralized market mechanism and get automatic conversion to TFTv2 end of the year.: 69.39%
I want my TFTA to be available on supported exchanges as TFT and agree with minimal price protection (0.15 USD, +2% increase per month starting with May 1), sales will happen through a sales bot.: 29.59%
I want my TFTA to be available on supported exchanges as TFT and there should be no price protection. I do realize this choice has the potential to damage the ThreeFold movement.: 1.02%
```
